### ISSUE TITLE
please tag and version this project

### ISSUE THREAD

**Issue by aviau** [nu7hatch/gouuid#29]
Hello,

Can you please tag and version this project?

I am the Debian Maintainer for gouuid and versioning would help Debian keep up with development.

**Comment by aviau** [comment]
Any news on this?

**Comment by dfuentes77** [comment]
Yes, this is so that when using a dependency manager, we can lock to a [semantic version](http://semver.org/) and get predictable builds

Otherwise, we're sticking to a commit sha or always pulling master.

For example, using glide, we can set to only pull certain patch/minor versions based on our comfort.

**Comment by nim-nim** [comment]
We'd like it too @  Fedora Linux

**Comment by jshube2** [comment]
Would still like to see this happen
